website/docs/plugin/framework/migrating/attributes-blocks/validators-predefined.mdx

```diff
@@ -0,0 +1,181 @@
+---
+page_title: 'Attribute predefined validators: Migrating from SDKv2 to the Framework'
+description: >-
+  Validations check required syntax, types, and acceptable values.
+  Migrate the predefined ConflictsWith, ExactlyOneOf, AtLeastOneOf and RequiredWith validators to the Framework.
+---
+
+# Validators - Predefined
+
+Attribute validators ensure that attributes do or do not contain specific values. You can use predefined validators for
+many use cases, or implement custom validators. Refer to [Schemas - Validators](/plugin/framework/schemas#validators) in
+the Framework documentation for details. Refer to the
+[Attributes - Custom Validators](/plugin/framework/migrating/validators-custom) page in this guide to learn how to
+implement custom validators.
+
+This page explains how to migrate a predefined validator from SDKv2 to the Framework.
+
+## SDKv2
+
+In SDKv2, the `ConflictsWith`, `ExactlyOneOf`, `AtLeastOneOf`, and `RequiredWith` fields on an attribute's
+`schema.Schema` struct perform predefined validations on the list of attributes set for these fields.
+
+```go
+func resourceExample() *schema.Resource {
+    return &schema.Resource{
+        /* ... */
+        Schema: map[string]*schema.Schema
+        }{
+            "attribute_example": {
+                ConflictsWith:  []string
+                ExactlyOneOf:   []string
+                AtLeastOneOf:   []string
+                RequiredWith:   []string
```

Comment:
Not familiar with this syntax - should these lines end with `,`?

Reply:
Fixed